lib/hammer_cli_katello/id_resolver.rb

```diff
@@ -93,23 +93,17 @@ def content_view_version_id(options)
 
       return options[key_id] if options[key_id]
 
-      begin
-        options[key_environment_id] ||= lifecycle_environment_id(
-          scoped_options("environment", options)
-        )
-      rescue HammerCLIForeman::MissingSeachOptions # rubocop:disable HandleExceptions
+      if options[key_environment_id]
+        options[key_environment_id] ||= search_and_rescue(
+          :lifecycle_environment_id, "environment", options)
         # Intentionally suppressing the exception,
         # environment is not always required.
       end
 
-      begin
-        options[key_content_view_id] ||= content_view_id(
-          scoped_options("content_view", options)
-        )
-      rescue HammerCLIForeman::MissingSeachOptions # rubocop:disable HandleExceptions
-        # Intentionally suppressing the exception,
-        # content_view is not always required.
-      end
+      options[key_content_view_id] ||= search_and_rescue(
+       :content_view_id, "content_view", options)
+      # Intentionally suppressing the exception,
```

Comment:
since the exception handling is now in search_and_rescue, we can probably move this comment there